src/plugins/messageLogger/index.tsx

```diff
@@ -220,6 +233,23 @@ export default definePlugin({
             ignoreGuilds.includes(ChannelStore.getChannel(message.channel_id)?.guild_id);
     },
 
+    EditMarker({ message, className, children, ...props }: any) {
+        return (
+            <span
+                {...props}
+                className={classes("messagelogger-edit-marker", className)}
+                onClick={() => openHistoryModal(message)}
+                aria-role="button"
+            >
+                {children}
+            </span>
+        );
+    },
+
+    Messages: proxyLazyWebpack(() => ({
```

Comment:
proxyLazyWebpack is for when you have webpack search inside your proxyLazy factory

outside webpack modules are already tracked anyway

```suggestion
    Messages: proxyLazy(() => ({
```

Reply:
[ಠ_ಠ](https://github.com/Vendicated/Vencord/pull/2299#discussion_r1542040744)